File: Objetivo 7/ExcecaoJava/src/model/Objeto.java
package model;

/**
 *
 * @author vagner
 */
public class Objeto {
    private String atributoA;
    private String atributoB;

    public Objeto(String atributoA, String atributoB) {
        this.atributoA = atributoA;
        this.atributoB = atributoB;
    }

    public String getAtributoA() {
        return atributoA;
    }

    public void setAtributoA(String atributoA) {
        this.atributoA = atributoA;
    }

    public String getAtributoB() {
        return atributoB;
    }

    public void setAtributoB(String atributoB) {
        this.atributoB = atributoB;
    }

    @Override
    public String toString() {
        return "Objeto{" + "atributoA=" + atributoA + ", atributoB=" + atributoB + '}';
    }
    
}

File: Objetivo5/ExcecaoPHP/model/Objeto.php
<?php

/**
 * Descreva aqui a classe Objeto.
 *
 * @author vagner
 */
class Objeto {
    private $atributoA;
    private $atributoB;
    
    function __construct($atributoA, $atributoB) {
        $this->atributoA = $atributoA;
        $this->atributoB = $atributoB;
    }

    public function getAtributoA() {
        return $this->atributoA;
    }

    public function getAtributoB() {
        return $this->atributoB;
    }

    public function setAtributoA($atributoA) {
        $this->atributoA = $atributoA;
    }

    public function setAtributoB($atributoB) {
        $this->atributoB = $atributoB;
    }

    public function __toString() {
        return "\nObjeto[Atributo A: " . $this->atributoA . " Atributo B: " . $this->atributoB . "]";
    }

}
